### PR TITLE
Issue #2109: Canceling the deletion of a role redirects to the "Manag…

### PR DESCRIPTION
**Pull request by klonos**
…e user accounts" page instead of back to "Roles" page.

Fixes https://github.com/backdrop/backdrop-issues/issues/2109

### CODE REVIEW

**Comment by klonos** [comment]
Random test failure?

**Comment by klonos** [comment]
...closing and reopening to start tests again.

File: core/modules/user/tests/user.test [hunk 1]
```diff
@@ -2120,6 +2120,13 @@ class UserRoleAdminTestCase extends BackdropWebTestCase {
     // Set this as the administrator role.
     config_set('system.core', 'user_admin_role', $admin_role->name);
 
+    // Test cancel deleting a role.
+    $role_name = 'administrator';
+    $this->backdropGet("admin/config/people/roles/delete/$role_name");
+    $this->clickLink(t('Cancel'));
+    $this->assertResponse(200);
+    $this->assertUrl('admin/config/people/roles', array(), 'Redirected to correct URL after cancel.');
```
Comment (quicksketch) [inline core/modules/user/tests/user.test:2128]: :+1:

**Comment by quicksketch** [comment]
Excellent. Merged into 1.x and 1.4.x.

**Pull request merged**